src/javascripts/components/example.mjs

```diff
@@ -1,22 +1,27 @@
 import iFrameResize from 'iframe-resizer/js/iframeResizer.js'
 
+/**
+ * Example component
+ *
+ * This allows automatic resizing of the iFrame pages contained within Example
+ * template wrappers.
+ *
+ * @param {Element} $module - HTML element to use for example
+ */
 function Example ($module) {
   this.$module = $module
```

Comment:
If it helps with the [comment above](https://github.com/alphagov/govuk-design-system/pull/2941/files#r1261272398) we'd do this on `govuk-frontend`

i.e. Accept an `Element` (like you've done) but return early if it's not an `HTMLIFrameElement`



```suggestion
  if (!($module instanceof HTMLIFrameElement)) {
      return this
    }

  this.$module = $module
```